src/components/NcInputField/NcInputField.vue

```diff
@@ -235,6 +235,13 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		/**
+		 * Class to add to the root component.
+		 */
+		class: {
+			type: [Object, String],
+			default: '',
+		},
```

Comment:
Array is also a valide `class` value

```suggestion
		class: {
			type: [Object, String, Array],
			default: '',
		},
```